kiota_serialization_json/json_parse_node.py

```diff
@@ -198,7 +198,7 @@ def get_enum_value(self, enum_class: K) -> Optional[K]:
         try:
             return enum_class[camel_case_key]  # type: ignore
         except KeyError:
-            raise Exception(f'Invalid key: {camel_case_key} for enum {enum_class}.')
+            return None
```

Comment:
Good idea, but we do not have project-wide logging yet, I think it would be good to have a project-wide logger and pass the logging info to it instead of adding one here